edit-post/assets/stylesheets/_z-index.scss

```diff
@@ -29,6 +29,17 @@ $z-layers: (
 	'.edit-post-header': 30,
 	'.wp-block-image__resize-handlers': 1, // Resize handlers above sibling inserter
 
+	// Should have lower index than anything else positioned inside the block containers
+	'.editor-block-list__block-drag-inset': 0,
+	// Should have higher index than anything else positioned inside the block containers
+	'.editor-block-list__block-drag-inset.is-visible > .editor-block-list__block-draggable-inner': 1000000000,
+
+	'.components-draggable__clone': 1000000000,
+
+	// Should have higher index than the inset/underlay used for dragging
```

Comment:
Why?

Reply:
I believe the drag inset was blocking some controls/buttons used for uploading images. Can you try this with a lower index than `'.editor-block-list__block-drag-inset'` above to confirm?

Reply:
Ok so I tried removing this entirely and I can't see any issue at the moment.

Reply:
Actually it's needed :) for the gray background when dragging 🤕

Reply:
The z-index itself is not really needed.

Reply:
Yeah, i'd be a bit cautious removing these indexes, cause they may end up blocking something not so obvious. I don't think I just sprinkled styles.

Reply:
> The z-index itself is not really needed. 

I do remember the drag inset blocking a button to upload an image in the image blocks.

Reply:
Since we're not showing the toolbars right now in the cloned component, might be it's not needed anymore. I can't find the issue myself.

Reply:
Could be. The issue wasn't related with the "clone", but with the actual panel that shows up when you add an image block but want to click to upload instead of drag an image from the desktop. Not sure if this was removed too. Let me load up Gutenberg to confirm then.

Reply:
>Yeah, i'd be a bit cautious removing these indexes

Conversely, we should be cautious with _adding_ them as well, and as made obvious by this thread, very clearly document their intention.

Reply:
I can click the buttons (Upload and Add media from Library) properly. Was this the issue?

Reply:
> Conversely, we should be cautious with adding them as well, and as made obvious by this thread, very clearly document their intention.

Absolutely. I didn't mean for my answer to excuse my clumsiness in adding these - I just didn't know any better. :-)

Reply:
> I can click the buttons (Upload and Add media from Library) properly. Was this the issue?

Ok. So the relation is still there. If I set `'.editor-block-list__block-draggable'` to a higher index than the other two, I cannot click or interact with the controls. I just confirmed with the latest version. But you probably right in saying that "The z-index itself is not really needed." since it defaults to `0`. Not sure why I had to be explicit about this.